compyle/profile.py

```diff
@@ -0,0 +1,100 @@
+""" Utils for profiling kernels
+"""
+
+import time
+from contextlib import contextmanager
+from collections import defaultdict
+from .config import get_config
+
+
+_profile_info = defaultdict(lambda: {'calls': 0, 'time': 0})
+
+
+def record_profile(name, time):
```

Comment:
Seems like this is now an implementation detail so this can be a private function,`_record_profile`.